tests/oss-fuzz/build.sh

```diff
@@ -54,13 +54,13 @@ if [ "$SANITIZER" == "memory" ]
 then
   export DAV1D_EXTRA_FLAGS="${DAV1D_EXTRA_FLAGS} -Denable_asm=false"
 fi
-sed -i 's/meson setup \(.*\) ../meson setup \1 '"${DAV1D_EXTRA_FLAGS}"' ../g' ./ext/dav1d.cmd
+sed -i 's/meson setup \(.*\) \.\./meson setup \1 '"${DAV1D_EXTRA_FLAGS}"' ../g' ./ext/dav1d.cmd
 
 # Build libaom with sanitizer flags.
 # Adds extra flags: -DAOM_TARGET_CPU=generic for msan.
 if [ "$SANITIZER" == "memory" ]
 then
-  sed -i 's/cmake \(.*\) ../cmake \1 -DAOM_TARGET_CPU=generic ../g' ./ext/aom.cmd
+  sed -i 's/cmake \(.*\) \.\./cmake \1 -DAOM_TARGET_CPU=generic ../g' ./ext/aom.cmd
```

Comment:
Without this change, this sed script also modifies the comment line `: # cmake and ninja must be in your PATH.`

```
diff --git a/ext/aom.cmd b/ext/aom.cmd
index 8d34ab3..fa5f718 100755
--- a/ext/aom.cmd
+++ b/ext/aom.cmd
@@ -3,7 +3,7 @@
 
 : # The odd choice of comment style in this file is to try to share this script between *nix and win32.
 
-: # cmake and ninja must be in your PATH.
+: # cmake and ninja must be in your -DAOM_TARGET_CPU=generic ..TH.
 
 : # If you're running this on Windows, be sure you've already run this (from your VC2019 install dir):
 : #     "C:\Program Files (x86)\Microsoft Visual Studio\2019\Professional\VC\Auxiliary\Build\vcvars64.bat"
@@ -14,6 +14,6 @@ cd aom
 mkdir build.libavif
 cd build.libavif
 
-cmake -G Ninja -DBUILD_SHARED_LIBS=OFF -DCONFIG_PIC=1 -DCMAKE_BUILD_TYPE=Release -DENABLE_DOCS=0 -DENABLE_EXAMPLES=0 -DENABLE_TESTDATA=0 -DENABLE_TESTS=0 -DENABLE_TOOLS=0 ..
+cmake -G Ninja -DBUILD_SHARED_LIBS=OFF -DCONFIG_PIC=1 -DCMAKE_BUILD_TYPE=Release -DENABLE_DOCS=0 -DENABLE_EXAMPLES=0 -DENABLE_TESTDATA=0 -DENABLE_TESTS=0 -DENABLE_TOOLS=0 -DAOM_TARGET_CPU=generic ..
 ninja
 cd ../..
```

I think it is because ` ..` matches the last space character and two arbitrary characters, so ` ..` matches ` PA`.